tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -46,6 +48,21 @@ _COMPILATION_LEVELS = {
   "advanced": ["--compilation_level=ADVANCED"]
 }
 
+_LANGUAGE_LEVELS = {
```

Comment:
Kamil: if you can merge it, would be awesome :)

On Wed, Feb 17, 2016 at 10:13 PM Michael Zhou <notifications@github.com>
wrote:

> In tools/build_rules/closure/closure_js_binary.bzl
> <https://github.com/bazelbuild/bazel/pull/910#discussion_r53232161>:
>
> > @@ -46,6 +48,21 @@ _COMPILATION_LEVELS = {
> >    "advanced": ["--compilation_level=ADVANCED"]
> >  }
> >
> > +_LANGUAGE_LEVELS = {
>
> I see :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bazelbuild/bazel/pull/910/files#r53232161>.
>